### PR TITLE
Feat/admin UI

### PR DESCRIPTION
**Pull request by Paribesh01**
https://github.com/user-attachments/assets/68f20ade-3989-45d5-bede-2e68e3756643

#547

### CODE REVIEW

**Comment by Paribesh01** [comment]
@VineeTagarwaL-code done with the work

**Comment by VineeTagarwaL-code** [comment]
payment history wasnt needed, and the recruiters are the ones with hr role right ?

**Comment by Paribesh01** [comment]
@VineeTagarwaL-code payment history was there in the screenshot you provided, yes the recruiters are the one with hr role.

**Comment by VineeTagarwaL-code** [comment]
Remove the payment history rest are good

**Comment by Paribesh01** [comment]
@VineeTagarwaL-code removed payment history

File: src/components/DeleteDialog.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
   DialogDescription,
   DialogFooter,
 } from './ui/dialog';
-import { ArchiveRestore, Trash } from 'lucide-react';
+import { ArchiveRestore, Trash2 } from 'lucide-react';
```
Comment (VineeTagarwaL-code) [inline src/components/DeleteDialog.tsx:16]: use icon file we have and import it from there, do the favour for archiveRestore too

File: src/config/path.config.ts [hunk 2]
```diff
@@ -5,7 +5,10 @@ const APP_PATHS = {
   SIGNUP: '/signup',
   RESET_PASSWORD: '/reset-password',
   JOBS: '/jobs',
-  MANAGE_JOBS: '/manage',
+
+  PAYMENT_HISTORY: '/manage/payments',
```
Comment (VineeTagarwaL-code) [inline src/config/path.config.ts:9]: why is manage payment code existing here ?

**Review by VineeTagarwaL-code — CHANGES_REQUESTED** [review]

**Comment by VineeTagarwaL-code** [comment]
Seems full of bugs here, fix it 
https://github.com/user-attachments/assets/adfd87e6-c446-48a7-bc37-bd1a780ca7fa

**Comment by Paribesh01** [comment]
@VineeTagarwaL-code ok I will work on changes, also for the recruiters, should I fetch users from db with hr role ??

**Comment by Paribesh01** [comment]
@VineeTagarwaL-code hr should have some relation with the company they are managing, so should i create a company model which have relation with the hr

**Comment by Paribesh01** [comment]
https://github.com/user-attachments/assets/feae8b1d-3e9d-4b50-8b01-23f0aa9567ae

@VineeTagarwaL-code done with the changes

**Comment by VineeTagarwaL-code** [comment]
Have you fixed that bug ? its still replicating

**Comment by Paribesh01** [comment]
https://github.com/user-attachments/assets/de38e512-198c-479b-9e42-0aa651a0df9b

@VineeTagarwaL-code its working fine..

**Comment by VineeTagarwaL-code** [comment]
Screen Shot 2024-10-31 at 00 17 02
everything works, fix the mobile responsiveness of this

**Comment by Paribesh01** [comment]
@VineeTagarwaL-code  done 
Screenshot 2024-10-31 022507

**Comment by VineeTagarwaL-code** [comment]
can you do the same for recruiters page ? and dont squeeze it too much just minimal space between the borders of the screen will work

**Comment by Paribesh01** [comment]
@VineeTagarwaL-code done

**Comment by VineeTagarwaL-code** [comment]
Great pr, make sure you on post on x and tag me !

**Pull request merged**